checkov/terraform/checks/resource/aws/StateMachineXray.py

```diff
@@ -0,0 +1,18 @@
+from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceValueCheck
+from checkov.common.models.enums import CheckCategories
+
+
+class StateMachineXray(BaseResourceValueCheck):
+
+    def __init__(self):
+        name = "Ensure State Machine has X-Ray tracing enabled"
+        id = "CKV_AWS_284"
+        supported_resources = ['aws_sfn_state_machine']
+        categories = [CheckCategories.LOGGING]
```

Comment:
I don't think this is logging

Reply:
not perfect, but the most appropriate one, we also categories the other X-ray related checks as `LOGGING`